http/headers/Cross-Origin-Opener-Policy.json

```
@@ -40,6 +40,44 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "noopener-allow-popups": {
+          "__compat": {
+            "description": "noopener-allow-popups support",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy",
+            "spec_url": "https://html.spec.whatwg.org/multipage/browsers.html#coop-noopener-allow-popups",
+            "support": {
+              "chrome": {
+                "version_added": "131"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": {
+                "version_added": false
+              },
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": false,
```

As per our data guidelines, a feature is considered [experimental if it's only supported by one browser engine](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#setting-experimental).

```suggestion
              "experimental": true,
```

---

I'm in the process of [landing the feature](https://github.com/WebKit/WebKit/pull/30344) in WebKit and it's about to land in the HTML spec. At the same time, I don't know what Safari version it would land in. Should this be considered "experimental"?